parquet-cli/src/main/java/org/apache/parquet/cli/commands/ToAvroCommand.java

```diff
@@ -96,9 +97,8 @@ public int run() throws IOException {
 
     Path outPath = qualifiedPath(outputPath);
     FileSystem outFS = outPath.getFileSystem(getConf());
-    if (overwrite && outFS.exists(outPath)) {
```

Comment:
SchemaCommand has the same problem.

Instead of throwing an exception here if the file already exists, how about adding a new method to BaseCommand for no overwrite case, and using that when creating the file in this command (and in SchemaCommand)?

Something like this: a new createWithNoOverwrite, which calls the private create with an extra overwrite=false flag, and it will use fs.create with this parameter value. How does this sound? We can also remove this delete file section in this case, and just call createWithNoOverwrite when overwrite is true, and call create otherwise.

Reply:
Thank you for the review @nandorKollar, that sounds good to me.
I'll update the PR a bit later.

Reply:
@nandorKollar Sorry for my long silence, just addressed your comments :)